core/templates/dev/head/pages/admin-page/jobs-tab/admin-jobs-tab.directive.html

```diff
@@ -76,25 +76,26 @@ <h4>Current time: <[$ctrl.HUMAN_READABLE_CURRENT_TIME]></h4>
 </md-card>
 
 <md-card ng-if="$ctrl.UNFINISHED_JOB_DATA.length > 0"
-         class="oppia-page-card oppia-long-text" style="max-width: 900px">
+         class="oppia-page-card oppia-long-text protractor-test-unfinished-jobs-card" style="max-width: 900px">
   <h3>Unfinished jobs</h3>
   <em>Note: This table may be stale; refresh to see the latest state.</em>
   <table class="table">
     <tr>
-      <th>Job ID</th>
+      <th class="protractor-test-unfinished-one-off-jobs-id-header">Job ID</th>
       <th>Status</th>
       <th>Time started</th>
       <th>Time finished</th>
       <th></th>
     </tr>
-    <tr ng-repeat="job in $ctrl.UNFINISHED_JOB_DATA track by $index">
-      <td><[job.id]></td>
+    <tr ng-repeat="job in $ctrl.UNFINISHED_JOB_DATA track by $index" class="protractor-test-unfinished-one-off-jobs-rows">
+      <td class="protractor-test-unfinished-one-off-jobs-id"><[job.id]></td>
       <td><[job.status_code]></td>
       <td><[job.human_readable_time_started]></td>
       <td><[job.human_readable_time_finished]></td>
       <td>
         <button ng-if="job.can_be_canceled"
-                ng-click="$ctrl.cancelJob(job.id, job.job_type)">
+                ng-click="$ctrl.cancelJob(job.id, job.job_type)"
+                class="float-right protractor-test-one-off-jobs-stop-btn">
```

Comment:
Just wondering, why did you add float-right here when it wasn't there before?

(I noticed there is a similar construct in line 42 too, so I guess the question is why we're adding it in both places as opposed to removing it from both places.)

Reply:
I didn't add it, it was present earlier on develop and was later removed. I have removed this but float-right on line 42 is present on develop and hence I am leaving it as it is.